src/thorlabs_cube/driver/kcube/kpa.py

```diff
@@ -0,0 +1,181 @@
+import struct as st
+
+from thorlabs_cube.driver.base import _Cube
+from thorlabs_cube.driver.message import MGMSG, SUBMSG, Message, MsgError
+
+_CHANNEL: int = 0x01
+_RESERVED: int = 0x00
+
+
+class Kpa(_Cube):
```

Comment:
```suggestion
class Kpa(Tpa):
```
This inheritance is desired, right? The class also needs to be imported.